src/main/java/au/gov/ga/geodesy/port/adapter/geodesyml/StringAdapter.java

```diff
@@ -0,0 +1,29 @@
+package au.gov.ga.geodesy.port.adapter.geodesyml;
```

Comment:
It should go into src/main/java/au/gov/ga/geodesy/support/marshalling/moxy/. The rough guide is business-level modelling into au.gov.ga.geodesy.port.adapter.geodesyml and supporting code into src/main/java/au/gov/ga/geodesy/support/...